Cargo.toml

```diff
@@ -24,6 +24,7 @@ futures = "0.3.28"
 jsonrpsee = "0.16.2"
 log = "0.4.17"
 once_cell = { version = "1.17.1", default-features = false }
+paste = "1.0"
 proptest = "1.1"
```

Comment:
I think these are `dev-dependencies`...?

Reply:
You probably have a misunderstanding for workspace deps compared to crate deps

In the workspace, you state versioning and feature gates, then use whatever you want (deps, dev-deps) in crate Cargo.toml.

So I'm using it as a dev dependency in runtime/mainnet/Cargo.toml